### PR TITLE
[backport 2.9] Add vGPU allocatable warning banner

### PR DESCRIPTION
**Pull request by torchiaf**
This is a backport for https://github.com/rancher/dashboard/pull/11017

Fixes https://github.com/rancher/dashboard/issues/11517

### CODE REVIEW

File: shell/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -1358,6 +1358,8 @@ cluster:
         macAddress: Mac Address
         macFormat: 'Invalid MAC address format.'
       vGpus:
+        warnings:
+          minimumAllocatable: It's highly recommended to select a vGPU with a number of allocatable devices greater then the number of nodes (Machine Count) to avoid "un-schedulable" errors after cluster updates.
```
Comment (gaktive) [inline shell/assets/translations/en-us.yaml:1362]: Small typo: `then` should be *than*

**Review by jordojordo — APPROVED** [review]

**Comment by gaktive** [comment]
@a110605 let us know if there's anything else for you to look at here since with Francesco out, we'll need someone else to merge. If you feel everything's OK based on your own concerns, then you can approve & merge. Code freeze is tomorrow for 2.8.next1 & 2.9.next1 so time is tight.

**Comment by nwmac** [comment]
@gaktive I spoke with @a110605 and this is awaiting backend changes- moving to 2.9.next1

**Comment by gaktive** [comment]
cc @rebeccazzzz 

@a110605 @nwmac just so it's written down somewhere, what backend changes are needed for the updates to the banner wording?

**Review by a110605 — APPROVED** [review]
Need rebase after https://github.com/rancher/dashboard/pull/12137 merge.

**Pull request merged**